rank/data/tests/images.ts

```diff
@@ -117,26 +117,31 @@ const tests: Test[] = [
         query: 'monsters',
         ratings: ['n7r5s65w'],
         description: "shouldn't match 'Monastery'",
+        knownFailure: true,
```

Comment:
The idea of labelling tests as 'known failures' feels funky to me, even if it is the way that we've operated IRL. Conscious that these labels might rot if we're not paying close attention.

We've also talked in the past about using an overall threshold score to let the system pass as a whole. I can see that adding known failures is simpler to implement than a set of thresholds, but i'm not sure what will be best long term. What's your feeling about this stuff at the moment?

Reply:
Yea, it felt odd. Let's see how we feel about uncommenting it when we add the new collections stuff.